FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -129,65 +134,69 @@ - (void)signInTesterWithCompletion:(void (^)(NSError *_Nullable error))completio
+    [request setURL:[NSURL URLWithString:URLString]];
+    [request setHTTPMethod:@"GET"];
+    [request setValue:[NSString stringWithFormat:@"Bearer %@", accessToken]
+        forHTTPHeaderField:@"Authorization"];
+
+    NSURLSessionDataTask *listReleasesDataTask = [URLSession
+        dataTaskWithRequest:request
+          completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
+            if (error) {
+              // TODO: Reformat error into error code
+              completion(nil, error);
+              return;
+            }
+
+            NSHTTPURLResponse *HTTPResponse = (NSHTTPURLResponse *)response;
+
+            if (HTTPResponse.statusCode == 200) {
+              [self handleReleasesAPIResponseWithData:data completion:completion];
+            } else {
+              // TODO: Handle non-200 http response
+              NSLog(@"ERROR - Non 200 service response - %@", HTTPResponse);
+              @throw([NSException exceptionWithName:@"NotImplementedException"
+                                             reason:@"This code path is not implemented yet"
+                                           userInfo:nil]);
+            }
+          }];
 
-      // perform your API request using the tokens
-        NSURLSession *URLSession = [NSURLSession sharedSession];
-        NSMutableURLRequest *request = [[NSMutableURLRequest alloc] init];
-        NSString *URLString =
-            [NSString stringWithFormat:kReleasesEndpointURL, [[FIRApp defaultApp] options].googleAppID];
-        [request setURL:[NSURL URLWithString:URLString]];
-        [request setHTTPMethod:@"GET"];
-        [request setValue:[NSString
-                              stringWithFormat:@"Bearer %@", accessToken]
-            forHTTPHeaderField:@"Authorization"];
-
-        NSURLSessionDataTask *listReleasesDataTask = [URLSession
-            dataTaskWithRequest:request
-              completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
-                if (error) {
-                  // TODO: Reformat error into error code
-                  completion(nil, error);
-                  return;
-                }
-
-                NSHTTPURLResponse *HTTPResponse = (NSHTTPURLResponse *)response;
-
-                if (HTTPResponse.statusCode == 200) {
-                  [self handleReleasesAPIResponseWithData:data completion:completion];
-                } else {
-                  // TODO: Handle non-200 http response
-                  NSLog(@"ERROR - Non 200 service response - %@", HTTPResponse);
-                  @throw([NSException exceptionWithName:@"NotImplementedException"
-                                                 reason:@"This code path is not implemented yet"
-                                               userInfo:nil]);
-                }
-              }];
-
-        [listReleasesDataTask resume];
-    }];
+    [listReleasesDataTask resume];
+  }];
```

Comment:
JFYI: The formatter doesn't seem to play well with blocks (seems to horrendously indent them). Since this is double indented it might fly off the page after you run `style.sh`.

I sometimes resolve this by moving the whole block into a method.

Reply:
Will follow up in a subsequent PR to this branch.